backend/polls/views.py

```diff
@@ -201,25 +182,21 @@ def statistics(self, *args, **kwargs):
             time_to_next_poll = (tomorrow - datetime.now()).seconds
```

Comment:
Instead of adding 1 day (24 hours), it should select the next day at 00:00. Currently I'm not sure the cache is renewed often enough

Reply:
I think that's already what I'm doing (see previous lines). Tomorrow is day + 1 at 12am.

Reply:
Indeed, my bad